Gems/Compression/Code/Include/Compression/DecompressionInterfaceAPI.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/std/containers/span.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <Compression/CompressionInterfaceStructs.h>
+
+namespace Compression
+{
+    enum class DecompressionResult : AZ::u8
+    {
+        Started,
+        Complete,
+        NotStarted,
```

Comment:
Nitpick: Consider `Pending` instead of `NotStarted` to avoid possible double negatives like `if (result != DecompressionResult::NotStarted)`.

Reply:
"Pending" doesn't really indicated whether it is pending start or pending completion.
How about "PendingStart"?